src/corefx/System.Globalization.Native/icushim.cpp

```diff
@@ -170,11 +170,31 @@ bool FindLibWithMajorMinorSubVersion(int* majorVer, int* minorVer, int* subVer)
     return false;
 }
 
-// GlobalizationNative_LoadICU
-// This method get called from the managed side during the globalization initialization. 
-// This method shouldn't get called at all if we are running in globalization invariant mode
-// return 0 if failed to load ICU and 1 otherwise
-extern "C" int32_t GlobalizationNative_LoadICU()
+#ifdef OSX_PLATFORM
```

Comment:
None of the functions / version constants above are needed on OSX, could you please ifdef them out?